projects/common/src/utilities/browser/storage/abstract-storage.ts

```diff
@@ -1,17 +1,36 @@
 import { Observable, Subject } from 'rxjs';
 import { distinctUntilChanged, filter, map, startWith } from 'rxjs/operators';
+import { DictionaryStorageImpl } from './dictionary-storage-impl';
 
 // A small abstraction on browser's storage for mocking and cleaning up the API a bit
 export abstract class AbstractStorage {
   private readonly changeSubject: Subject<string> = new Subject();
```

Comment:
is this redundant ?

Reply:
Redundant with what? Not sure I follow - if you mean the type portion, we have a lint rule that requires all class members to have a declared type.

Reply:
I meant, this `changeSubject`. I saw we're using `changeSubject.next` but apart from that this is not being used, or am I missing something?

Reply:
It's used in `watch` - it's not visible by default in this PR because it didn't change (neither did the subject, just the line below it 😉 )